src/views/activity/activity.view.tsx

```diff
@@ -33,6 +33,7 @@ const Activity: FC = () => {
   const [lastLoadedItem, setLastLoadedItem] = useState(0);
   const [total, setTotal] = useState(0);
   const [wrongNetworkBridges, setWrongNetworkBridges] = useState<Bridge["id"][]>([]);
+  const [finaliseDisabledBridges, setFinaliseDisabledBridges] = useState<Bridge["id"][]>([]);
```

Comment:
"Finalise disabled bridges" as a name of a variable sounds quite weird (in fact it sounds like an action). I propose to rename it to just `disabledBridges`, simpler and easy to read. At the end it's just a list of ids from bridges. Another alternative could be `disabledBridgeActions`, but I prefer the first one.

Reply:
You're right. I read it as "finalise-disabled bridges" but in camel case, it looks like an action. Done.